icubam/backoffice/server.py

```diff
@@ -0,0 +1,45 @@
+import os.path
+
+import tornado.ioloop
+import tornado.locale
+import tornado.web
+from absl import logging
+from tornado_sqlalchemy import SQLAlchemy
+
+from icubam.backoffice.handlers.home import HomeBOHandler
+
+
+class BackOfficeServer:
+  """Serves and manipulates the Backoffice ICUBAM."""
+
+  def __init__(self, config, port):
+    self.config = config
+    self.port = port
+    self.routes = []
+    self.db = SQLAlchemy("sqlite+pysqlite:///{}".format(self.config.db.sqlite_path))
```

Comment:
then merely use `sqlite.SQLDB(self.config.db.sqlite_path)`. This way someone can improve the db without having to touch this part of code.
But again, I fully agree, it is probably meaningful to look at the tornado_sqlalchemy package